### ISSUE TITLE
Codeception Error: Creating default object from empty value

### ISSUE THREAD

**Issue by davist11** [engram-design/SuperTable#403]
**Description**

We have <a href="https://craftcms.com/docs/3.x/testing/">automated testing</a> setup for our site and everything has been running smoothly. I just added Super Table, and now I'm getting an error when running the test suite.

```
1) ErpApiControllerCest: No authorization header
 Test  tests/functional/ErpApiControllerCest.php:noAuthorizationHeader
                                                                      
  [yii\base\ErrorException] Creating default object from empty value  
                                                                      

Scenario Steps:

 1. $I->amOnPage("/actions/erp/api/hello") at tests/functional/ErpApiControllerCest.php:13

/var/www/html/vendor/craftcms/cms/src/web/ErrorHandler.php:76
/var/www/html/vendor/verbb/base/src/BaseHelper.php:39
/var/www/html/vendor/verbb/super-table/src/base/PluginTrait.php:61
/var/www/html/vendor/verbb/super-table/src/SuperTable.php:58
/var/www/html/vendor/yiisoft/yii2/base/BaseObject.php:109
/var/www/html/vendor/yiisoft/yii2/base/Module.php:158
/var/www/html/vendor/craftcms/cms/src/base/Plugin.php:115
/var/www/html/vendor/yiisoft/yii2/di/Container.php:420
/var/www/html/vendor/yiisoft/yii2/di/Container.php:171
/var/www/html/vendor/yiisoft/yii2/BaseYii.php:365
/var/www/html/vendor/craftcms/cms/src/services/Plugins.php:925
/var/www/html/vendor/craftcms/cms/src/test/CraftConnector.php:104
/var/www/html/vendor/codeception/module-yii2/src/Codeception/Lib/Connector/Yii2.php:599
/var/www/html/vendor/codeception/module-yii2/src/Codeception/Lib/Connector/Yii2.php:319
/var/www/html/vendor/symfony/browser-kit/AbstractBrowser.php:384
/var/www/html/vendor/codeception/lib-innerbrowser/src/Codeception/Lib/InnerBrowser.php:233
/var/www/html/vendor/codeception/module-yii2/src/Codeception/Module/Yii2.php:697
/var/www/html/vendor/codeception/lib-innerbrowser/src/Codeception/Lib/InnerBrowser.php:281
/var/www/html/vendor/codeception/lib-innerbrowser/src/Codeception/Lib/InnerBrowser.php:393
/var/www/html/vendor/codeception/module-yii2/src/Codeception/Module/Yii2.php:680
/var/www/html/vendor/codeception/codeception/src/Codeception/Step.php:268
/var/www/html/vendor/codeception/codeception/src/Codeception/Scenario.php:76
/var/www/html/tests/_support/_generated/FunctionalTesterActions.php:530
/var/www/html/tests/functional/ErpApiControllerCest.php:13
/var/www/html/vendor/codeception/codeception/src/Codeception/Lib/Di.php:128
/var/www/html/vendor/codeception/codeception/src/Codeception/Test/Cest.php:138
/var/www/html/vendor/codeception/codeception/src/Codeception/Test/Cest.php:150
/var/www/html/vendor/codeception/codeception/src/Codeception/Test/Cest.php:82
/var/www/html/vendor/codeception/codeception/src/Codeception/Test/Test.php:98
/var/www/html/vendor/phpunit/phpunit/src/Framework/TestSuite.php:677
/var/www/html/vendor/codeception/phpunit-wrapper/src/Runner.php:117
/var/www/html/vendor/codeception/codeception/src/Codeception/SuiteManager.php:161
/var/www/html/vendor/codeception/codeception/src/Codeception/Codecept.php:208
/var/www/html/vendor/codeception/codeception/src/Codeception/Codecept.php:162
/var/www/html/vendor/codeception/codeception/src/Codeception/Command/Run.php:514
/var/www/html/vendor/codeception/codeception/src/Codeception/Command/Run.php:408
/var/www/html/vendor/symfony/console/Command/Command.php:256
/var/www/html/vendor/symfony/console/Application.php:971
/var/www/html/vendor/symfony/console/Application.php:290
/var/www/html/vendor/symfony/console/Application.php:166
/var/www/html/vendor/codeception/codeception/src/Codeception/Application.php:117
/var/www/html/vendor/codeception/codeception/app.php:46
/var/www/html/vendor/codeception/codeception/app.php:47
/var/www/html/vendor/codeception/codeception/codecept:7
```

Looks like it's blowing up because of this line: https://github.com/verbb/super-table/blob/bc35da9a3a2eef20338c1e60998c4afd57f84851/src/base/PluginTrait.php#L61

**Steps to reproduce**
1. Setup Craft test suite
2. Install Super Table
3. Run test suite

**Additional info**
- Plugin version: 2.6.7
- Craft version: Craft Pro 3.6.6
- Multi-site: No

**Comment by davist11** [comment]
Just fixed this in my `tests/_craft/config/app.php` by adding

```php
'components' => [
    'log' => null,
],
```

Not sure on the implications of that long term, but it at least prevents this error.

**Comment by engram-design** [comment]
I believe I've fixed this in [verbb-base 1.0.3](https://github.com/verbb/verbb-base/releases/tag/1.0.3). A `composer update` should resolve this updated dependancy.

**Comment by davist11** [comment]
Sweet yeah that fixed it. Thanks!
